sdks/python/tests/library_integration/metrics_with_llm_judge/test_evaluation_metrics.py

```diff
@@ -4,6 +4,10 @@
 from ...testlib import assert_helpers
 import langchain_openai
 from opik.evaluation.models.langchain import langchain_chat_model
+from ragas.metrics import ExactMatch, LLMContextPrecisionWithoutReference
```

Comment:
please import modules instead of names, ideally in a way like
`from ... import as ragas_...` otherwise it's hard to understand while reading the code where does the class come from (similar for langchain)